ui/pages/send/send-header/send-header.component.test.js

```diff
@@ -39,7 +39,7 @@ describe('SendHeader Component', () => {
           history: { mostRecentOverviewPage: 'activity' },
         }),
       );
-      expect(getByText('Add Recipient')).toBeTruthy();
+      expect(getByText('Send to')).toBeTruthy();
     });
 
     it('should render "Send" for DRAFT stage when asset type is NATIVE', () => {
```

Comment:
Do we want to keep this existing behavior or change this as well?

Reply:
This is fine to keep